src/sentry/api/bases/dashboard.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+from rest_framework import serializers
+from sentry.models import WidgetDisplayTypes
+
+
+class DashboardSerializer(serializers.Serializer):
+    title = serializers.CharField(required=True)
+    # TODO(lb): Can't figure out how to represent a json blob.
```

Comment:
Hmm... @billyvg or @lynnagara do you happen to know how to represent a json blob with our version of the django rest framework?

Reply:
`JSONField` maybe?